packages/common/src/types/metricQuery.ts

```diff
@@ -143,4 +144,5 @@ export type MetricQueryRequest = {
     customDimensions?: CustomDimension[];
     granularity?: DateGranularity;
     metadata?: MetricQuery['metadata'];
+    timezone?: string;
```

Comment:
I'm planning to make this an Enum with the picker checkin. Any concerns?

Reply:
yeah, that makes sense, I didn't know what values to give, so I used a generic string